src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -1,6 +1,10 @@
 import findLastIndex from 'ramda/src/findLastIndex';
 import propSatisfies from 'ramda/src/propSatisfies';
 import path from 'ramda/src/path';
+import webLogger from '#lib/logger.web';
```

Comment:
Should we be using `nodeLogger from '#lib/logger.node';` instead here? You can then use the logger in this format:

logger.error(RADIO_SCHEDULE_DATA_INCOMPLETE_ERROR, { error });

I think webpack swaps out the node logger for the web logger when it is bundled for the client - this is going to be run on both!

This code will be run both on the server and on the client - I think

Reply:
I kept the web logger in the radio schedules component in the use effect, as it is also used in most read and the toggles container. A conversation for after this should we just be using one, standardised logger, and letting webpack handle the swap?